### ISSUE TITLE
Package the main application into an executable binary

### ISSUE THREAD

**Issue by gridhead** [t0xic0der/wisdom-cli#4]
Make sure that you use PyInstaller and that it is compliant to Python 3.8 and above.

**Comment by gridhead** [comment]
Would you be willing to pick this up, @khallnayak?

**Comment by Sameer-Mann** [comment]
@t0xic0der i would like to help with this. Though it my first time. I am reading the documentation of pyinstaller.

**Comment by Sameer-Mann** [comment]
@t0xic0der i tried to make an executable for the app but it requires me sign the code manually on mac [https://github.com/pyinstaller/pyinstaller/issues/4934](https://github.com/pyinstaller/pyinstaller/issues/4934) and even if i did that the binary file would not be distributable.

So i think using pyinstaller to distribute a mac os package is not feasible correct me if i am wrong.

**Comment by gridhead** [comment]
Sure, @Sameer-Mann. I am assigning this issue to you.

Feel free to take your time and learn about PyInstaller.

**Comment by gridhead** [comment]
Yes. The issue seems to be troubling especially macOS users. Could you try making an executable on a GNU/Linux system?

**Comment by Sameer-Mann** [comment]
ok I will try to make it work for gnu/linux systems

**Comment by Sameer-Mann** [comment]
@t0xic0der i made the package but it gives this when i try to run it.
![Screenshot 2020-11-04 at 2 35 35 PM](https://user-images.githubusercontent.com/42488833/98091186-0c53ee00-1eab-11eb-9614-e28ca7de65b6.png)
After i export those two variables it works fine though.
The encoding which the script is using and which it should use are not the same. 
Could u tell me how to add encoding based on the environment.
I tried reading https://click.palletsprojects.com/python3/ but could not figure out how to do it.

**Comment by gridhead** [comment]
The code environment for the project is UTF-8 compliant only and does not support ASCII, unless added a exporting variable. 

But then again, the export is restricted to your programming environment only and should not affect the repository.

Further reading : https://click.palletsprojects.com/en/7.x/python3/#python-3-surrogate-handling

**Comment by Sameer-Mann** [comment]
okay so now what should i do?
